platform/config/config-model-api/src/main/java/org/codice/ddf/config/model/ConfluenceSourceConfig.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.config.model;
+
+import java.util.stream.Stream;
+
+public interface ConfluenceSourceConfig extends SourceConfig {
+
+  String getUsername();
```

Comment:
❗️ We should start documenting those and documenting what is optional and what is not and how we behave if not specified.

Reply:
Documentation is on the configuration abstraction wiki.